rules.js

```diff
@@ -8,11 +8,9 @@ module.exports = {
     'color-no-invalid-hex': true,
     'font-family-name-quotes': 'always-unless-keyword',
     'font-weight-notation': 'numeric',
-    'media-feature-no-missing-punctuation': true,
-    'declaration-block-no-ignored-properties': true,
     'number-leading-zero': 'never',
-    'selector-no-empty': true,
     'string-quotes': 'single',
+    'rule-empty-line-before': ['always', { 'except': 'first-nested'} ],
```

Comment:
You're right, but I tested on `lexdoit` and no errors were found.
Fixed anyway!